### DISCUSSION TITLE
Question about rgdal and gdal dependencies

### DISCUSSION THREAD

**Discussion by tlcaputi** [r-spatial/sf#2432] [General]
Hi everyone,

I'm not a system admin and am well out of my wheelhouse -- but I'll do my best to communicate my problem.

I'm trying to have my system admin install the dependencies that would allow me to install the "sf" R package. However, he states that:

1. "sf" depends on rgdal, which has been retired and is no longer maintained
2. "sf" is compatible only with gdal35 or less, and that is not compatible with libarmadillo 12.x, which others need

Does anyone know of a way around this?

**Comment by rsbivand** [comment]
Both claims are false. `sf` runs with GDAL 3.9.2, the current release, and has run with all earlier versions, down to the earliest supported. It has never required `rgdal`, nor any packages requiring retired packages. Documentation inter alia on https://r-spatial.github.io/evolution/. @tlcaputi you also need to state your platform (Windows, macOS, if Linux which kind, in all cases which version), and which version of R.
